extensions/amazon-lambda/runtime/pom.xml

```diff
@@ -23,7 +23,6 @@
         <dependency>
             <groupId>org.junit.jupiter</groupId>
             <artifactId>junit-jupiter</artifactId>
-            <scope>test</scope>
```

Comment:
Was this changed by mistake?

Reply:
hell....good catch...Yes a mistake.

Reply:
Mistakingly put it in runtime instead of the integration test pom